editor/import/editor_scene_importer_gltf.cpp

```diff
@@ -29,7 +29,7 @@
 /*************************************************************************/
 
 #include "editor_scene_importer_gltf.h"
-
```

Comment:
Keep the separation line, it's from the style guide: https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Reply:
Kept. Thanks for linking to the style guide.